photutils/isophote/harmonics.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import (absolute_import, division, print_function, unicode_literals)
+
+import numpy as np
+from scipy.optimize import leastsq
```

Comment:
@ibusko A lot of the functionality in `photutils` requires `scipy` and `scikit-image`, but they are "optional" dependencies.  In other words, `import photutils` will still work without those dependencies.  However, the actual functions that need those packages will raise an `ImportError` if they are not installed.  This mirrors the behavior in `astropy`, where `numpy` is the only "strict" dependency.

@bsipocz Perhaps we should just make `scipy` and `scikit-image` required dependencies?  I don't recall if affiliated packages are not supposed to have required dependencies other than `numpy` and `astropy`.